docs/VictoriaLogs/data-ingestion/Fluentd.md

```diff
@@ -0,0 +1,109 @@
+---
+weight: 2
+title: Fluentd setup
+disableToc: true
+menu:
+  docs:
+    parent: "victorialogs-data-ingestion"
+    weight: 2
+aliases:
+  - /VictoriaLogs/data-ingestion/Fluentd.html
+  - /victorialogs/data-ingestion/fluentd.html
+  - /victorialogs/data-ingestion/Fluentd.html
+---
+VictoriaLogs supports given below Fluentd outputs:
+- [Loki](#loki)
+- [HTTP JSON](#http)
+
+## Loki
+
+Specify [loki output](https://docs.fluentd.io/manual/pipeline/outputs/loki) section in the `fluentd.conf`
+for sending the collected logs to [VictoriaLogs](https://docs.victoriametrics.com/victorialogs/):
+
+```conf
+<match **>
+  @type loki
+  url "http://localhost:9428/insert"
+  <buffer>
+    flush_interval 10s
+    flush_at_shutdown true
+  </buffer>
+  custom_headers {"VL-Msg-Field": "log", "VL-Time-Field": "time", "VL-Stream-Fields": "path"}
```

Comment:
`"VL-Stream-Fields": "path"` what value `path` contains? Is it a good candidate for stream field?

Reply:
tail input has no ability to add extra metadata without custom lua scripts. replaced it with fluentd docker driver
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7123/files